src/Components/MovieDetails/MovieDetails.js

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
-import movieData from '../sample-data';
 import './MovieDetails.css';
 
 const MovieDetails =({ movieInfo }) => {
-  const { poster_path, title, average_rating, release_date, backdrop_path } = movieInfo; 
+  const { poster_path, title, average_rating, release_date, backdrop_path, overview, budget, revenue, genres, tagline, runtime } = movieInfo; 
```

Comment:
Glad you did all this destructuring on it's own line, rather than in the props. Much more legible